pkg/adapter/adapter_test.go

```diff
@@ -689,6 +704,10 @@ func TestAdapter_msgContentType(t *testing.T) {
 			if tt.err && err == nil {
 				t.Errorf("Unexpected error state for msg type want:\n%+s\ngot:\n%+s", tt.want, got)
 			}
+
```

Comment:
I wonder if the L700 above should be:

```
if err == nil && got != tt.want {
```
When there's no error we should check if got == want or am I missing something?

Reply:
Here im checking if there is an error, but yeah the error shouldnt have the tt.want but the error message let me change that :)

Reply:
I'm saying that L700 checks if got != tt.want only if there's an error? That seems wrong? msgContentType returns err == nil if all goes well, and that's when we should be testing if the got == tt.want, not when there's an error like what we're doing here?